### PR TITLE
add interop tests to rust-libp2p required tests.

### PR DESCRIPTION
**Pull request by jxs**
### What else do we need to know?

**DRI:** myself

### Reviewer's Checklist

- [ ] It is clear where the request is coming from (if unsure, ask)
- [ ] All the automated checks passed
- [ ] The YAML changes reflect the summary of the request
- [ ] The Terraform plan posted as a comment reflects the summary of the request

### CODE REVIEW

**Review by MarcoPolo — APPROVED** [review]

**Review by mxinden — APPROVED** [review]
:pray:

File: github/libp2p.yml [hunk 1]
```diff
@@ -4308,6 +4308,7 @@ repositories:
             - Compile on x86_64-pc-windows-msvc
             - Compile with select features (mdns tcp dns async-std)
             - Compile with select features (mdns tcp dns tokio)
+            - Interoperability Testing
```
Comment (thomaseizinger) [inline github/libp2p.yml:4311]: This is the name of the workflow, not the name of the job.

Your workflow currently has two jobs but only the latter is likely the one we want to make required.

In the following screenshot, you can see the `Compile on wasm32-unknown-unknown` job that is mentioned on line 4305 above. There is no mention which workflow it is in (i.e. the `Continuous integration` isn't mentioned in here).

image

Interestingly enough, the actual test job has another level of nesting: https://github.com/libp2p/rust-libp2p/actions/runs/3962950188/jobs/6791796862

My guess would be that we should be changing this to:

```suggestion
            - Run testplans test
```

**Review by thomaseizinger — CHANGES_REQUESTED** [review]
Thanks for submitting this PR! I think there is a small mistake.

File: github/libp2p.yml (hunk 1 above)
Comment (jxs) [reply github/libp2p.yml]: Pff, thanks for this Thomas! :pray: Updated

**Pull request merged**